2025년 2월 정기배포 가이드 (클라) (#363)
TOAST-DOCS/Gamebase: en/release-notes-unreal.md

## Game > Gamebase > Release Notes > Unreal

### 2.69.1 (2025. 3. 4.)
[SDK Download](https://static.toastoven.net/toastcloud/sdk_download/gamebase/v2.69.1/GamebaseSDK-Unreal.zip)

#### 기능 추가

* (Windows) 런칭 정보에서 약관 정보를 확인할 수 있도록 추가했습니다.
    * FGamebaseLaunchingInfo::FApp::FTermsService

#### 기능 개선/변경

> * API 호출 시 매개변수로 전달받는 `UGamebaseJsonObject`를 `FGamebaseVariantMap(TMap&lt;FName, FVariant&gt;)`으로 변경했습니다.

Author: ```suggestion
* API 호출 시 매개변수로 전달 받는 `UGamebaseJsonObject`를 `FGamebaseVariantMap(TMap&lt;FName, FVariant&gt;)`으로 변경했습니다.
```

> * 내부 로직을 개선하였습니다.

Author: ```suggestion
* 내부 로직을 개선했습니다.
```

#### 버그 수정

> * (Windows) 게스트 로그인에서 UUID 발급과정 오류로 인해 모두 동일한 값이 생성되는 문제를 수정했습니다.

Author: ```suggestion
* (Windows) 게스트 로그인 시 UUID 발급 과정 오류로 인해 모두 동일한 값이 생성되는 문제를 수정했습니다.
```

> ### 2.69.1 (2025. 3. 4.)
> [SDK Download](https://static.toastoven.net/toastcloud/sdk_download/gamebase/v2.69.1/GamebaseSDK-Unreal.zip)
> 
> #### 기능 추가
> 
> * (Windows) 런칭 정보에서 약관 정보를 확인할 수 있도록 추가했습니다.
>     * FGamebaseLaunchingInfo::FApp::FTermsService
> 
> #### 기능 개선/변경
> 
> * API 호출 시 매개변수로 전달 받는 `UGamebaseJsonObject`를 `FGamebaseVariantMap(TMap&lt;FName, FVariant&gt;)`으로 변경했습니다.
> * 내부 로직을 개선했습니다.
> 
> #### 버그 수정
> 
> * (Windows) 게스트 로그인 시 UUID 발급 과정 오류로 인해 모두 동일한 값이 생성되는 문제를 수정했습니다.
> * (Windows) Line IDP 로그인 시 region 설정이 동작하지 않는 문제를 수정했습니다.
> * (Windows) 킥아웃 시 ServerPushAppKickOut 이벤트 발생과 팝업이 노출되지 않는 문제를 수정했습니다.
> * (Windows) 심볼 생성 시 엔진의 Build Configuration이 Development가 아닌 경우 오류가 발생하는 문제를 수정했습니다.
> * (Android) 환경에 따라 RegisterPush가 동작하지 않는 문제를 수정했습니다.
> 
> #### 플랫폼별 변경 사항

Reviewer: ```suggestion
### 2.69.1 (March 4, 2025)
[SDK Download](https://static.toastoven.net/toastcloud/sdk_download/gamebase/v2.69.1/GamebaseSDK-Unreal.zip)

#### Added Features

* (Windows) Added terms and conditions information so you can check it in the launch information.
    * FGamebaseLaunchingInfo::FApp::FTermsService

#### Feature Updates

* Changed `UGamebaseJsonObject` passed as a parameter when calling API to `FGamebaseVariantMap(TMap&lt;FName, FVariant&gt;)`.
* Improved internal logic.

#### Bug Fixes

* (Windows) Fixed an issue where all UUIDs generated during guest login were the same due to an error in the UUID issuance process.
* (Windows) Fixed an issue where region settings did not work when logging in to Line IDP.
* (Windows) Fixed an issue where the ServerPushAppKickOut event would not trigger and the popup would not be displayed when kicked out.
* (Windows) Fixed an issue where an error would occur when generating symbols if the engine's Build Configuration was not Development.
* (Android) Fixed an issue where RegisterPush would not work depending on the environment.

#### Platform-specific Changes
```
